Author: Andreas Beckmann <anbe@debian.org>
Description: prepend /usr/lib/#PRIVATE#/modules to ModulePath
 The alternate solution, putting the ModulePath directive in the
 OutputClass section, only works with xserver 1.20 onwards and is
 therefore not backwards-compatible to stretch.

--- a/NVIDIA-Linux/nvidia-drm-outputclass.conf
+++ b/NVIDIA-Linux/nvidia-drm-outputclass.conf
@@ -4,6 +4,11 @@
 # version 3.9 or higher with CONFIG_DRM enabled, and only if the nvidia-drm.ko
 # kernel module is loaded before the X server is started.
 
+Section "Files"
+    ModulePath     "/usr/lib/#PRIVATE#/modules"
+    ModulePath     "/usr/lib/xorg/modules"
+EndSection
+
 Section "OutputClass"
     Identifier     "nvidia"
     MatchDriver    "nvidia-drm"
